classes/04class/exercises/c04-iac02/zarajoy/output.tf

```diff
@@ -0,0 +1,16 @@
+output "subnet-public-1" {
+  value = ["${aws_subnet.subnet-public-1.cidr_block}"]
```

Comment:
Just be aware that interpolation only is [deprecated](**url**) so try to avoid them.

Reply:
:P removed the interpolation from the outputs.